templates/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ genrule(
     name = "templates-tarball",
     srcs = glob([
         "skeleton/**",
+        "migrate/**",
```

Comment:
Are users meant to be able to run `daml new myproject migrate` to create a project based off the `migrate` template? What does that mean?

If not, it might be better to package this separately, not as part of the templates tarball.

Reply:
I think that would actually be fine. If a user wants to create his migration path manually, that's ok. We just provide the tools to automate it.

Reply:
👍